docs/concepts/correlated-objects-pattern.md

```diff
@@ -41,31 +41,30 @@ In the simplest case, you are only writing to members of a collection object, no
 > It's generally a good practice to put have a final `context.sync` just before the closing "}" character of the application `run` method (such as `Excel.run`, `Word.run`, etc.). This is because the `run` method makes a hidden call of `context.sync` as the last thing it does if, and only if, there are queued commands that have not yet been synchronized. The fact that this call is hidden can be confusing, so we generally recommend that you add the explicit `context.sync`. However, given that this article is about minimizing calls of `context.sync`, it is actually more confusing to add an entirely unnecessary final `context.sync`. So, in this article, we leave it out when there are no unsynchronized commands at the end of the `run`.
 
 ```javascript
-Word.run(async function (context) {
-    let startTime, endTime;
-    const docBody = context.document.body;
-
-    // search() returns an array of Ranges.
-    const searchResults = docBody.search('the', { matchWholeWord: true });
-    context.load(searchResults, 'items');
-    await context.sync();
-
-    // Record the system time.
-    startTime = performance.now();
-
-    for (var i = 0; i < searchResults.items.length; i++) {
-      searchResults.items[i].font.highlightColor = '#FFFF00';
-
-      await context.sync(); // SYNCHRONIZE IN EACH ITERATION
-    }
-    
-    // await context.sync(); // SYNCHRONIZE AFTER THE LOOP
-
-    // Record the system time again then calculate how long the operation took.
-    endTime = performance.now();
-    console.log("The operation took: " + (endTime - startTime) + " milliseconds.");
-  })
-}
+await Word.run(async function (context) {
+  let startTime, endTime;
```

Comment:
This is a petty thing, but do we have a standard on 2 versus 4 spaces for code sample indents?